apps/aecore/c_src/Makefile

```diff
@@ -100,6 +99,10 @@ help: ## This documentation
 
 all: deps $(LIB_SO) ## Build all
 
+# Use 256-times smaller memory footprint for nodes/edges in order to speed up unit tests
+test: all
+      CXXFLAGS += -DEDGEBITS=19
```

Comment:
A comment on the choice of number `19` would help - both for readability and for clarifying whether overflow is possible if e.g. an argument from the Erlang (test) invocation is changed.